Resources/Prototypes/_RPSX/Entities/Clothing/OuterClothing/coats.yml

```
@@ -8,3 +8,17 @@
     sprite: _RPSX/Clothing/OuterClothing/Coats/adminitrator_coat.rsi
   - type: Clothing
     sprite: _RPSX/Clothing/OuterClothing/Coats/adminitrator_coat.rsi
+
+- type: entity
+  parent: ClothingOuterCoatDetectiveLoadoutGrey
+  id: ClothingOuterCoatDeathCloak
```

Почему это и Coat и Cloack одновременно? Либо 
```suggestion
  id: ClothingOuterCoatDeath
```
Либо
```suggestion
  id: ClothingOuterCloakDeath
```
Но во 2 случае надо переместить в отдельный файлик.